webapp/views.py

```diff
@@ -64,7 +64,15 @@ def _build_mirror_list():
         flask.request.headers.get("X-Real-IP", flask.request.remote_addr)
     )
 
-    if ip_location and "country" in ip_location:
+    if not local:
```

Comment:
Instead of the `if` and `elif`. I'd just do a quick return of the `mirror_list` inside each if. But it's up to you.